tests/config.js

```diff
@@ -43,7 +43,7 @@ api_secret=abc
       let writeFileSync = fs.writeFileSync;
       fs.writeFileSync = function(filename, data){
         expect(filename).to.match(/\/\.nexmorc$/);
-        expect(data).to.equal(ini_content);
+        expect(data.replace(/\r\n/gm, '\n')).to.equal(ini_content);
```

Comment:
Tested on Windows and this works. However, instead of using replace, could you modify this to use the `EOL` constant provided in the `os` module?

Something like: `const EOL = require('os').EOL;` this will allow the system to select the appropriate line ending each time.

:)